.github/workflows/django.yml

```diff
@@ -0,0 +1,55 @@
+name: Djangoplicity Customsearch CI
+
+on:
+  push:
+    branches: [ master, develop ]
+  pull_request:
+    branches: [ master, develop ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      fail-fast: false
+      matrix:
+        python-version: [2.7, 3.7, 3.8]
+
+    services:
+      postgres:
+        image: postgres:10.5
+        env:
+          POSTGRES_DB: djangoplicity-customsearch
+          POSTGRES_USER: djangoplicity-customsearch
+          POSTGRES_PASSWORD: djangoplicity-customsearch
+        ports:
+          - 5432:5432
+        # needed because the postgres container does not provide a healthcheck
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: psycopg2 prerequisites
+        run: sudo apt-get install python-dev libpq-dev
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+          pip install pytest-django
+      - name: Run migrations
+        run: python manage.py migrate
+      - name: Run tests
+        run: |
+          coverage run --source='.' manage.py test
+          coverage xml
```

Comment:
If you are only using here Django tests `(manage.py test)`, then why would you need pytest?.